docs/source/child_services.rst

```diff
@@ -148,13 +148,13 @@ See services communicate in real time: running the child services template
 
 1. Contact Octue to request a Google Cloud Platform service account credentials file.
 
-2. Save this file locally and create a ``GCP_SERVICE_ACCOUNT`` environment variable whose value is the file's absolute path. This variable must be available to all three terminal windows used to run the template - see below for one method of doing this. **IMPORTANT**: Do not commit this or any other credentials or credentials file to git, GitHub, or any other version control software or website - doing so opens you, your systems and equipment, and our systems and equipment up to hackers and cyber attack.
+2. Save this file locally and create a ``GOOGLE_APPLICATION_CREDENTIALS`` environment variable whose value is the file's absolute path. This variable must be available to all three terminal windows used to run the template - see below for one method of doing this. **IMPORTANT**: Do not commit this or any other credentials or credentials file to git, GitHub, or any other version control software or website - doing so opens you, your systems and equipment, and our systems and equipment up to hackers and cyber attack.
```

Comment:
I changed it because it's the same environment variable the Google SDK looks for, allowing us to use it as before but also opening it up to a wider variety of usages. I can't remember the exact place I needed this (it might have been the GitLab CI for `data-gateway`, but I needed to define `GOOGLE_APPLICATION_CREDENTIALS` to make it work, which pointed to the same thing `GCP_SERVICE_ACCOUNT` did anyway.